### ISSUE TITLE
[Nightly chatterino-windows-old-x86-64-Qt-5.15.2] The procedure entry point SetThreadDescription could not be located

### ISSUE THREAD

**Issue by UnfortunateLiama** [Chatterino/chatterino2#5543]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Caused by https://github.com/Chatterino/chatterino2/commit/1ccdaea8eefc75614467078ea9a7fa6801c21007

### Offtopic : 
```
MSVCP140.dll
MSVCP140_1.dll
MSVCP140_ATOMIC_WAIT.dll
VCRUNTIME140.dll
VCRUNTIME140_1.dll
``` 
will cause 

> The program can't start because api-ms-win-core-libraryloader-l1-2-0.dll is missing from your computer. Try reinstalling the program to fix this problem. 

deleting those dlls from chatterino folder will allow it to start without errors ¯\_(ツ)_/¯

### OS and Chatterino Version

Windows 7 Version 6.1 (Build 7601: SP 1), kernel: 6.1.7601

**Comment by pajlada** [comment]
Once the artifact is available (in about 20 minutes or so), can you try the build from https://github.com/Chatterino/chatterino2/pull/5544 ?

**Comment by Nerixyz** [comment]
Oh yea, my bad. We'd need to `LoadLibrary` on Qt 5, it's easier to only do it on Qt 6.

**Comment by pajlada** [comment]
Fixed by #5544
